tsconfig.json

```diff
@@ -3,6 +3,8 @@
     "include": ["projects/**/*"],
     "exclude": ["projects/**/*.spec.ts", "projects/demo-integrations", "**/schematics/**/*"],
     "angularCompilerOptions": {
+        "enableIvy": false,
+        "compilationMode": "full",
```

Comment:
<img width="869" alt="image" src="https://user-images.githubusercontent.com/12021443/163147954-4f303240-200c-45b7-9849-2cbe9f288521.png">

will explicitly indicate this property, it is already so by default